### ISSUE TITLE
Composite package icon

### ISSUE THREAD

**Issue by mbarakaja** [Templarian/MaterialDesign#1916]
I have an application where I use the [`package-variant-closed`](https://github.com/Templarian/MaterialDesign/blob/master/icons/svg/package-variant-closed.svg) icon in my list of products.

The list contains two type of product, simple and compound (product made up of other products) and I want to differentiate them.

So, is there a possibility to make an icon that looks like [Font Awesome's cubes](http://fontawesome.io/icon/cubes/) icons?.

So, the idea is to use the actual `package-variant-closed` icon and just triplicate it in a layout equal to the font awesome cube icon.

Thanks.

**Comment by mspoelstra** [comment]
++ For this icon also.

**Comment by Templarian** [comment]
![cubes](https://cloud.githubusercontent.com/assets/338885/23931761/d22397a2-0901-11e7-898d-61ecd1e3e39e.png) ![cubes48](https://cloud.githubusercontent.com/assets/338885/23931764/de514434-0901-11e7-8b08-931c8ddeee22.png)

These cube icons are not very Material Design. Hmm... 😕

**Comment by mspoelstra** [comment]
Agreed, but an icon to denote a bundle/composite-package would be fantastic! Cheers.

**Comment by CoDEmanX** [comment]
cube-unfolded, widgets, google-circles-communities come to mind.

Wasn't there an icon like the following somewhere already?

![1489566437_597624](https://cloud.githubusercontent.com/assets/1309249/23939687/7cc4527c-0994-11e7-9bff-edb595d79b93.png)

**Comment by mbarakaja** [comment]
As a temporary solution I'm using the [**`buffer icon`**](https://github.com/Templarian/MaterialDesign/blob/master/icons/svg/buffer.svg)
![download](https://cloud.githubusercontent.com/assets/7861175/23952568/8621f5b2-099a-11e7-8388-399debe5a06c.png)

Maybe that can give you some idea.

**Comment by PeterShaggyNoble** [comment]
`cube-unfolded` would have been my first suggestions as well. An alternative to using `buffer` might be `layers`, which comes with an `-off` variant if you needed it.

![](https://materialdesignicons.com/icon/layers/) ![](https://materialdesignicons.com/icon/layers-off/)

**Comment by mbarakaja** [comment]
I want to share with you some variations that I have tried.

A problem that I have is that I was not able to fit the icons in the 20x20 pixel template. Right now the smaller icons have a width of 21px. With that size (21px) the vertical lines fit pixel by pixel properly.

![composite-package-examples](https://cloud.githubusercontent.com/assets/7861175/23965515/e1493894-09c0-11e7-9f7e-2c149e4772de.png)

**Comment by mbarakaja** [comment]
I added one more variant

![composite-packages](https://cloud.githubusercontent.com/assets/7861175/23972095/2e70eb0e-09d9-11e7-99a5-d9df7456dd84.png)

**Comment by vizo** [comment]
Any updates on this?

**Comment by mbarakaja** [comment]
Why should be close this @JamesCoyle? Is there a good alternative already?

**Comment by goyney** [comment]
The suggestions above break [Material guidelines](https://material.io/design/iconography/system-icons.html#system-icon-metrics):

![Screen Shot 2019-07-28 at 8 15 30 PM](https://user-images.githubusercontent.com/1015040/62014874-7a701800-b174-11e9-8cf3-aadab8d04a5c.png)

---

Basically, we need another idea that is Material to depict what you're asking for. I'll leave this issue open for a little bit longer, but without any other new ideas, we may have to close it.

**Comment by goyney** [comment]
Going to close this issue due because there really doesn't seem to be another idea on how to do this in a Material way. If someone comes up with some other ideas, please don't be afraid to share! Thanks for the suggestion.

**Comment by CoDEmanX** [comment]
Looks like a pinwheel at native size:

![grafik](https://user-images.githubusercontent.com/1309249/62688686-a045ba00-b9c9-11e9-8c15-0f79777b92ad.png)

Some variants:

![grafik](https://user-images.githubusercontent.com/1309249/62690233-a5f0cf00-b9cc-11e9-9a89-fd66b456ae1c.png)

![grafik](https://user-images.githubusercontent.com/1309249/62690262-b3a65480-b9cc-11e9-8d8d-c2f5ff74afda.png)

![grafik](https://user-images.githubusercontent.com/1309249/62690279-bdc85300-b9cc-11e9-91e5-1b9df45ef032.png)

Stacked pyramid (chart icon?):

![grafik](https://user-images.githubusercontent.com/1309249/62688751-c1a6a600-b9c9-11e9-844b-01260de2acf0.png)

![grafik](https://user-images.githubusercontent.com/1309249/62688793-dd11b100-b9c9-11e9-9d80-74d4edad645b.png)

**Comment by MrGrigri** [comment]
What if you removed one of the puzzle pieces?

**Comment by goyney** [comment]
Reopening as closing it stirred up discussion. 😄

**Comment by CoDEmanX** [comment]
@MrGrigri You mean like this?

![grafik](https://user-images.githubusercontent.com/1309249/62736310-54c4f780-ba2d-11e9-86ae-244a1afde16e.png)

I like it!

Another variant would be to make the bottom two one:

![grafik](https://user-images.githubusercontent.com/1309249/62736531-d583f380-ba2d-11e9-956f-f4d34e0fb25f.png)

It makes it less versatile though.

What do you think about the connection variants? I tend towards `c`.

I think we should add the first pyramid as `chart-pyramid`.

**Comment by CoDEmanX** [comment]
Simplified connectors, but kinda blurry:

![grafik](https://user-images.githubusercontent.com/1309249/62737403-fc432980-ba2f-11e9-86fb-0f0cf5e6b732.png)

Making the hole side of each piece chevron like is a no-go, looks like a swastika in the end.

**Comment by mbarakaja** [comment]
> The suggestions above break Material guidelines:

@goyney So, doesn't [package-variant-closed](https://github.com/Templarian/MaterialDesign/blob/master/icons/svg/package-variant-closed.svg) brakes Material Design guidelines aswell?

**Comment by CoDEmanX** [comment]
This icon might actually work:

[![view-grid-plus](https://materialdesignicons.com/icon/view-grid-plus)](https://materialdesignicons.com/icon/view-grid-plus)

**Comment by goyney** [comment]
> > The suggestions above break Material guidelines:
> 
> @goyney So, doesn't [package-variant-closed](https://github.com/Templarian/MaterialDesign/blob/master/icons/svg/package-variant-closed.svg) brakes Material Design guidelines aswell?

Yes, it does. We allow breaks in very rare circumstances after internal discussions with collaborators. After playing around with multiple cubes within Material's guidelines, I do not think this is one we're going to consider adding.

![image](https://user-images.githubusercontent.com/1015040/131710157-9697de26-ea0c-4a2b-966a-1d041cb98735.png)![image](https://user-images.githubusercontent.com/1015040/131710185-71483f99-0a4d-4034-a6f5-49bb53d2c06a.png)

They just don't have any meaning at native size. (And even larger size, IMO.)

I believe the puzzle options are going to be the best bet.
